Modix.Bot/Modules/QuoteModule.cs

```diff
@@ -41,6 +41,14 @@ public class QuoteModule : ModuleBase
 
                 if (message == null)
                     message = await FindMessageInUnknownChannelAsync(messageId);
+
+                var user = Context.User as IGuildUser;
+                var channel = message.Channel as ITextChannel;
```

Comment:
Pretty minor, but I tend to prefer hard casts over `as`, because getting an `InvalidCastException` immediately gives you better information than getting a `NullReferenceException` later.

```suggestion
                var user = (IGuildUser)Context.User;
                var channel = (ITextChannel)message.Channel;
```